flycheck.el

```diff
@@ -453,24 +453,39 @@ commands through `bundle exec', `nix-shell' or similar wrappers."
   :package-version '(flycheck . "0.25")
   :risky t)
 
-(defcustom flycheck-executable-find #'executable-find
+(defcustom flycheck-executable-find #'flycheck-find-executable
```

Comment:
I'm not super happy with the names (we have `flycheck-executable-find`, `flycheck-find-executable`, and `executable-find`).  What do you think of `flycheck-default-executable-find`?